content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Setup Repo Info
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+*See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
+
+## Installing the Components
+
+### Installing Base Component
+
+The **Base** chart contains the basic CRDs and cluster roles required to set up Istio.
+This should be installed prior to any other Istio component.
+
+{{< text bash >}}
+$ helm install istio-base istio/base
+{{< /text >}}
+
+### Installing CNI Component
```

Comment:
Is CNI necessary for Ambient?  It isn't installed by istioctl install as far as I can tell....

Reply:
It should be installed with the `ambient` profile and yea it is.

Reply:
It is mandatory.. the CNI component handles all the redirection logic for both the node and pods. Without it, the pods won't be put into the ambient mesh.